v4/source/channels.yaml

```diff
@@ -37,7 +37,7 @@
                 description: Markdown-formatted text to display in the header of the channel
               type:
                 type: string
-                description: "'O' for a public channel, 'P' for a private channel"
+                description: "'O' for a public channel, 'P' for a private channel, 'D' for a direct channel"
```

Comment:
Does this API endpoint actually let you create a direct channel? I'm pretty sure it only let's you create private and public channels

Reply:
I think I saw the code as well, I will update the comment in a while https://github.com/search?q=org%3Amattermost+%27P%27+for+a+direct+channel&type=Code.

Reply:
It won't let you create direct channels, see:

https://github.com/mattermost/mattermost-server/blob/master/api4/channel.go#L76

which calls and will return an error here for using direct or group channels:

https://github.com/mattermost/mattermost-server/blob/master/app/channel.go#L146